automated-comments/go/general/golint_not_satisfied.md

```diff
@@ -0,0 +1,14 @@
+There's a great tool called [`golint`](https://github.com/golang/lint) which will examine the code for common problems and style issues.
+Try running `golint` on the solution, e.g.: `golint two_fer.go`; it will make some useful suggestions.
```

Comment:
```suggestion
Run `golint` on the solution, e.g.: `golint two_fer.go`; it will make some useful suggestions.
```

The problem with `try` is that, if it fails, a student might leave it. With an imperative it's more likely that they'll complain if it doesn't work. (I have no proof for this 😅, only anecdotal evidence).